dashboards-observability/common/constants/data_table.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const GRID_HEADER_COLUMN_MAX_WIDTH = '150px';
+export const GRID_PAGE_RANGE_DISPLAY = 5;
+export const COLUMN_DEFAULT_MIN_WIDTH = 100;
+export const GRID_PAGE_SIZES = [10, 50, 100];
+export const ROW_DENSITIES = [
+  { icon: 'tableDensityExpanded', height: 55, selected: false },
+  { icon: 'tableDensityNormal', height: 45, selected: false },
+  { icon: 'tableDensityCompact', height: 35, selected: true },
+];
+
+export const HEADER_HEIGHT = 35;
```

Comment:
If the data table is used inside event analytics let's move this to `constants/explorer.ts`, if shared by other components let's move to `constants/shard.ts`